GpsDataCapturer/sharedlibrary/src/main/java/com/google/sharedlibrary/LocationManagerListener.java

```diff
@@ -0,0 +1,44 @@
+package com.google.sharedlibrary;
+
+import android.location.GpsStatus;
+import android.location.Location;
+import android.location.LocationListener;
+import android.os.Bundle;
+
+/**
+ * This class implements LocationListener and GpsStatus.Listener callback functions and handle
+ * location updates and Gps status via Location Manager API
+ */
+public class LocationManagerListener implements LocationListener, GpsStatus.Listener {
+    private final String TAG = "GeneralLocationListener";
+    private GpsDataCaptureService gpsDataCaptureservice;
```

Comment:
this field should be final:
private final GpsDataCaptureService gpsDataCaptureservice;

Reply:
Done.